Jenkinsfile.central

```diff
@@ -62,13 +62,13 @@ node {
                         --recv-keys ${GPG_KEY_ID} &&
                     echo ${GPG_KEY} | base64 -d >signing_key.asc &&
                     gpg --import signing_key.asc &&
-                    ./sbt ";sonatypeOpen ${BUILD_NUMBER};publish;sonatypeRelease"
```

Comment:
Previously, we were used Jenkins' `BUILD_NUMBER` to help ensure uniqueness across Sonatype staging repositories. It doesn't seem like `sonatypeBundleRelease` does anything to replicate that on its own, unless we override `sonatypeSessionName`. Is that a concern?

Reply:
`sonatypeSessionName` works out to be something like `projectName version` by default, or `root 1.42.0` for the last release. 

`sonatypeBundleRelease` is an alias for `; sonatypePrepare; sonatypeBundleUpload; sonatypeRelease`. The first step for `sonatypePrepare` is to drop the any existing staging repository using `sonatypeSessionName` as a unique key.

This isn't a concern.